helm_deploy/values-dev.yaml

```diff
@@ -16,6 +16,11 @@ generic-service:
     PRISON-REGISTER_API_URL: https://prison-register-dev.hmpps.service.justice.gov.uk
     WHEREABOUTS_API_URL: https://whereabouts-api-dev.service.justice.gov.uk
     APPLICATIONINSIGHTS_CONFIGURATION_FILE: applicationinsights.dev.json
+    HMPPS_SQS_USE_WEB_TOKEN: "true"
+
+  namespace_secrets:
+    HMPPS_SQS_QUEUES_PRISONVISITSEVENTS_ARN: "topic_arn"
```

Comment:
do we not need the other properties in dev?

Reply:
I'm not sure HMPPS_SQS_QUEUES_PRISONVISITSEVENTS_QUEUE_NAME may be